.github/workflows/deployment.yml

```diff
@@ -55,14 +56,29 @@ jobs:
       - name: Set up JDK
         uses: actions/setup-java@v1
         with:
-          java-version: 14
+          java-version: 15-ea
```

Comment:
Why do we need to upgrade to 15?

Reply:
Because of this bug https://bugs.openjdk.java.net/browse/JDK-8238184